hack/make-rules/test-integration.sh

```diff
@@ -50,12 +50,17 @@ kube::test::find_integration_test_dirs() {
 cleanup() {
   kube::log::status "Cleaning up etcd"
   kube::etcd::cleanup
+  kube::log::status "Cleaning up Consul"
```

Comment:
Are we launching a whole consul too?  It almost seems like we need a breadcrumb on startup.